src/components/tracking-card/tracking-card.css

```
@@ -0,0 +1,55 @@
+@import url("https://fonts.googleapis.com/css2?family=Signika:wght@600&display=swap");
+
+.tracking-card {
+  position: relative;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr;
+  place-items: center;
+  overflow: hidden;
+  padding: 1rem 2rem 1rem 2rem;
```

You can shorten this :)
```suggestion
  padding: 1rem 2rem;
```